src/soap.js

```diff
@@ -16,6 +16,7 @@ function _requestWSDL(url, options, callback) {
     callback = options;
     options = {};
   }
+  _wsdlCache = options.WSDL_CACHE || _wsdlCache;
```

Comment:
How was `_wsdlCache` utilized before? can we move the variable declaration inside the function i.e. can we simplify this statement to `var _wsdlCache = options.WSDL_CACHE || {};`?

Reply:
`_wsdlCache` is currently being set outside of the `_requestWSDL` function, but is only being used there. From looking at the code the idea of `_wsdlCache` would be to that if `createClient` is called multiple times with the same `wsdl`, this `wsdl` would only need to be loaded the first time instead of every time. To move the initialization of `_wsdlCache` into the `_requestWSDL` function, this would mean that the cache was reset each time `createClient` is called (unless it is passed in via the options) and the benefit of caching would be lost.

Reply:
Thanks for clarifying @jonrober-80. That makes sense now :-).